src/mic/__main__.py

```diff
@@ -284,7 +311,7 @@ def step4(mic_file, configuration_files):
 )
 def step5(mic_file):
     """
-    Editing the MIC Wrapper and building your environment
+    Detect if code is executable, then set up Docker Image for model.
```

Comment:
I don't understand the new sentence. 

In this step you have to edit your mic wrapper and prepare your Docker image. What does `detect if code is executable` mean?

Reply:
I took that from the google docs. I should have written that more clearly, but I can just remove it since it doesnt effect the user's usage of the command

Reply:
ok!